@here/harp-mapview/test/MapViewTest.ts

```diff
@@ -105,6 +105,174 @@ describe("MapView", function() {
         expect(mapView.geoCenter.longitude).to.be.closeTo(coords.longitude, 0.000000000001);
     });
 
+    it("Correctly sets geolocation and zoom from options in constructor", function() {
```

Comment:
I don't think that these new tests proove that camera position is ok.
It's lot of boilerplate only to check that some (untested by the way) function is called.
Also, this test is very fragile to one particular implementation that we have now.

Instead,  I would cross-check, that map created with certain parameters reports proper position (even hardcoded in test) in `camera.possition` and `Utils.extractAttitude` yield expected results.

That would mean, we need to have two kinds of tests - for mercator and sphere projection.

Reply:
I improved the tests now. When using spherical projection the extracted attitude seems to be off by quite some bit (~0.001 degrees). I don't know if this is only caused by floating point imprecision or if there is more to it. For mercator projection the extracted attitude is very close.